controllers/NUgus_Controller/Makefile

```
@@ -65,6 +65,9 @@
 ###
 ###-----------------------------------------------------------------------------
 
+CFLAGS = -Wall
```

```suggestion
CFLAGS = -Wall -Wpedantic -Wextra -O3 -fPIC
CXXFLAGS = ${CFLAGS} -std=c++11
```
The standard is required for r-value references.

Not sure why we're using a makefile, but if we're setting flags here, these ones are good

---

Webots controllers is made with the provided makefile.

---

Probably better that we use cmake, maintain consistency across all of our repos as much as possible

---

Is it possible to include a `makefile.include` in cmake?

---

No, that is against the way cmake works

---

gist.github.com › gsc07
Webots Controller build using CMake · GitHub